app/src/main/java/com/asfoundation/wallet/ui/iab/PaymentMethodsAnalytics.kt

```diff
@@ -1,14 +1,12 @@
 package com.asfoundation.wallet.ui.iab
 
 import cm.aptoide.analytics.AnalyticsManager
-import com.asfoundation.wallet.analytics.AmplitudeAnalytics
 import com.asfoundation.wallet.analytics.RakamAnalytics
 import com.asfoundation.wallet.billing.analytics.BillingAnalytics
 
 class PaymentMethodsAnalytics(private val analyticsManager: AnalyticsManager,
                               private val billingAnalytics: BillingAnalytics,
-                              private val rakamAnalytics: RakamAnalytics,
-                              private val amplitudeAnalytics: AmplitudeAnalytics) {
+                              private val rakamAnalytics: RakamAnalytics) {
```

Comment:
We should inject the interface instead `analyticsSetup: AnalyticsSetup`, since we can have more than one implementation of this